reflex/utils/prerequisites.py

```diff
@@ -200,6 +200,39 @@ def initialize_gitignore():
         f.write(f"{(path_ops.join(sorted(files))).lstrip()}")
 
 
+def initialize_requirements_txt():
+    """Initialize the requirements.txt file if applicable."""
+    # If the requirements.txt file already exists, return.
```

Comment:
Comment may be out of date.

Can simplify this to 
* create a file if it doesn't exist
* then do the normal logic